selfdrive/pandad/panda.cc

```diff
@@ -62,8 +62,8 @@ void Panda::set_safety_model(cereal::CarParams::SafetyModel safety_model, uint16
   handle->control_write(0xdc, (uint16_t)safety_model, safety_param);
 }
 
-void Panda::set_alternative_experience(uint16_t alternative_experience) {
-  handle->control_write(0xdf, alternative_experience, 0);
```

Comment:
We went from sending 0 to sending something. I am guessing the 0 was params from stock?

Reply:
It used to send 0 because stock doesn't use this field with this endpoint, so it is freed up for us to send whatever we want via SPI/USB.